apps/rush/src/start.ts

```diff
@@ -1,38 +1,32 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
-// The minimal set of imports that are safe even for ancient NodeJS versions:
+// The minimal set of imports that are safe even for ancient Node.js versions:
 import * as colors from 'colors';
 import * as os from 'os';
 import * as semver from 'semver';
 
 const nodeVersion: string = process.versions.node;
-
-// tslint:disable-next-line
-
-// We are on an ancient version of NodeJS that is known not to work with Rush
-if (semver.satisfies(nodeVersion, '<= 6.4.0')) {
-  console.error(colors.red(`Your version of Node.js (${nodeVersion}) is very old and incompatible with Rush.`
-    + ` Please upgrade to the latest Long-Term Support (LTS) version.`));
+let alreadyReportedNodeTooNewError: boolean = false;
+
+// We are on an ancient version of Node.js that is known not to work with Rush
+if (semver.satisfies(nodeVersion, '< 8.9.0')) {
+  console.error(colors.red(
+    `Your version of Node.js (${nodeVersion}) is very old and incompatible with Rush. ` +
+    `Please upgrade to the latest Long-Term Support (LTS) version.`
+  ));
   process.exit(1);
 }
 
 // We are on a much newer release than we have tested and support
 // tslint:disable-next-line
-else if (semver.satisfies(nodeVersion, '>=11.0.0')) {
-  console.warn(colors.yellow(`Your version of Node.js (${nodeVersion}) has not been tested with this release of Rush.`
-    + ` The Rush team will not accept issue reports for it.`
-    + ` Please consider upgrading Rush or downgrading Node.js.`));
-}
-
-// We are not on an LTS release
-// tslint:disable-next-line
-else if (!semver.satisfies(nodeVersion, '^6.9.0')
-      && !semver.satisfies(nodeVersion, '^8.9.0')
-      && !semver.satisfies(nodeVersion, '^10.13.0')) {
-  console.warn(colors.yellow(`Your version of Node.js (${nodeVersion}) is not a Long-Term Support (LTS) release.`
```

Comment:
Seems like the "not LTS" message was removed entirely.  We need to re-add it in **rush-lib**.  Per my other comment, it should probably be performed /after/ rush.json has been loaded, so we can consider the `suppressNodeLtsWarning` option.